lib/jsdom/browser/file/blob.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+function Blob() {
+  if (!(this instanceof Blob)) {
+    throw new TypeError("DOM object constructor cannot be called as a function.");
+  }
+  var parts = arguments[0];
+  var properties = arguments[1];
+  if (arguments.length > 0) {
+    if (!parts || typeof parts !== "object" || parts instanceof Date || parts instanceof RegExp) {
+      throw new TypeError("blobParts should be nor a non-objects nor Dates nor RegExps nor null");
+    }
+  }
+
+  const buffers = [];
+
+  if (parts) {
+    var l = +parts.length;
+    for (var i = 0; i < l; i++) {
+      var part = parts[i];
+      var buffer;
+      if (part instanceof ArrayBuffer) {
+        buffer = new Buffer(new Uint8Array(part));
```

Comment:
I used the argument for the slice method and I forgot to change it. I think I could use Uint8Array but I don't have so much time in my life :), you still can change it later anyway.